### DISCUSSION TITLE
Add location rewrite nginx

### DISCUSSION THREAD

**Discussion by MohammedAlasaad** [laravel/valet#1134] [Q&A]
Hi
I just want to add new location rewrite but it seems not working on valet conf file in /usr/local/etc/nginx/valet/valet.conf. I saw ppl saying we need to add using Driver but not really sure how to make that. And these called with react fetch request. not in URL.
```
   location /api {
        rewrite api/app/(.*)$ /api/app/index.php?request=$1;
  }
```
Not sure why there is folder called Nginx and its empty in valet directory /Users/NAME/.config/valet/Nginx

Thanks

**Comment by ph00lt0** [comment]
@MohammedAlasaad did you get this to work? I have tried to apply this in many places but none of the rules seem to be picked up.
